itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -485,6 +485,9 @@ private Task getReplLoadRootTask(String sourceDb, String replicadb, boolean isIn
       metricCollector = new BootstrapLoadMetricCollector(replicadb, tuple.dumpLocation, 0,
         confTemp);
     }
+    /* When 'hive.repl.retain.custom.db.locations.on.target' is enabled, the first iteration of repl load would
+       run only database creation task, and only in next iteration of Repl Load Task execution, remaining tasks will be
+       executed. Hence disabling this to perform the test on task optimization.  */
```

Comment:
its also because the locations would overwrite and things like that right?

Reply:
For  this test case, it was just the task what it validates and hence have not added here.

Reply:
Why is this set to false in BaseReplicationScenarios